internal/rbd/controllerserver.go

```diff
@@ -97,20 +97,23 @@ func (cs *ControllerServer) parseVolCreateRequest(
 	req *csi.CreateVolumeRequest) (*rbdVolume, error) {
 	// TODO (sbezverk) Last check for not exceeding total storage capacity
 
-	isMultiNode := false
+	isMultiWriter := false
 	isBlock := false
 	for _, capability := range req.VolumeCapabilities {
 		// RO modes need to be handled independently (ie right now even if access mode is RO, they'll be RW upon attach)
-		if capability.GetAccessMode().GetMode() == csi.VolumeCapability_AccessMode_MULTI_NODE_MULTI_WRITER {
-			isMultiNode = true
+		// below capability check indicates that we support both {SINGLE_NODE or MULTI_NODE} WRITERs and the `isMultiWriter`
+		// flag has been set accordingly.
+		if (capability.GetAccessMode().GetMode() == csi.VolumeCapability_AccessMode_MULTI_NODE_MULTI_WRITER) ||
+			(capability.GetAccessMode().GetMode() == csi.VolumeCapability_AccessMode_SINGLE_NODE_MULTI_WRITER) {
```

Comment:
question:
if I read this currently we are not allowing rbd filesystem mode pvc on a single node with multiple pods `VolumeCapability_AccessMode_SINGLE_NODE_MULTI_WRITER`  is that correct?

RBD filesystem RWO should be supported?